### ISSUE TITLE
moving syntax to the top of the reference pages

### ISSUE THREAD

**Issue by tegacodes** [processing/p5.js-website#542]
### Increasing Access

It would make the syntax more prominent and easier to find for beginners.

### Most appropriate sub-area of p5.js?

Reference

### Feature request details

Hey y'all, really great job on the new site. It looks great. I think moving the syntax for each function to the top of each function page would make it easier to find. The p5 reference is often used to check syntax and having to scroll down the page and look for it can be time consuming.

**Comment by computationalmama** [comment]
Yes you are right something like the old website would be great. I've often found that it was easier to refer through it. 
Did you mean like this one @tegacodes?

<img width="796" alt="Screenshot 2024-09-24 at 12 39 25 AM" src="https://github.com/user-attachments/assets/96da8f80-f568-4621-9777-27d336039b56">

**Comment by limzykenneth** [comment]
@tegacodes Only just came across this but I think what you mean is similar to what is mentioned in #582? 

What I'm thinking is that people copy more often from the example for an immediately working code instead of the syntax definition, but maybe I'm wrong on this impression?

**Comment by davepagurek** [comment]
I'm sympathetic to putting the syntax at the top personally, I end up scrolling to the bottom most of the time. This is especially for functions like `image()` where the examples have a bunch of numbers and I'd like to see the names of the parameters to more easily differentiate between them. Not sure how common that is for other people though, so I don't have a super strong opinion on this.

**Comment by computationalmama** [comment]
**What I was thinking:**

For the main page: https://p5js.org/reference/

- All features on top gives creators a sense about the possibilities (for new creators)
- For returning creators/coders who just want a quick reference 

I'd love for it to be a bit more inviting to folks who are exploring and pushing their p5js journey forward!

> I end up scrolling to the bottom most of the time.

Like @davepagurek I too find that I have to scroll way down, but I don't mind it so much in on the internal pages: https://p5js.org/reference/p5/circle/

**Comment by limzykenneth** [comment]
We can possibly set up A/B testing for more evidence for either case but the setup of which probably would be a bit complicated, serving the two versions I can do it relatively easily with Cloudflare Workers but the data collection probably need a bit more.

**Comment by computationalmama** [comment]
Cool!  If it saves you some cycles @limzykenneth we could just ask a few folks in different groups? Don't know if that's a reasonable way to think about this stuff 🤔
